libs/core/__deprecated__/renderer/components/core/icon/icon.config.ts

```diff
@@ -508,6 +509,7 @@ const typeToIcon: Partial<Record<IconType, typeof Arrow>> = {
   [IconType.RecoveryModeWhite]: RecoveryModeWhite,
   [IconType.RecoveryModeBlack]: RecoveryModeBlack,
   [IconType.ButtonSuccess]: ButtonSuccess,
+  [IconType.LightButton]: LightButton,
```

Comment:
⚪ (project sugesstion)

What do you think about using a transparent background for better versatility?

A transparent background might help the icon adapt better across different backgrounds, keeping a cleaner visual consistency.